web/client/themes/default/less/geostory.less

```diff
@@ -245,7 +247,8 @@
         /* size of content */
         width: @ms-story-size-full;
     }
-    .ms-section-title {
+    .ms-section-title,
+    .ms-section-banner {
```

Comment:
I think this style is not needed because it applies to content style `.ms-content-text`